### PR TITLE
107 create stay left planner

### PR DESCRIPTION
**Pull request by selmaillig**
Implement base case - plan for area without obstacles. 
The first of four tests pass.

### CODE REVIEW

File: hlc/plan/planner.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        left_layer.append((layer, i))
+
+    for j in range(grid_dimension[1]):
+        right_layer.append(grid_dimension[1] - layer, j)
+
+    for k in range(grid_dimension[0]):
+        top_layer.append(k, grid_dimension[0] - layer)
+
+    for l in range(grid_dimension[0]):
+        bottom_layer.append(l, layer)
```
Comment (elli-wolf) [inline hlc/plan/planner.py:37]: This function is never called and useless as long as it doesn't return anything.

File: hlc/plan/planner.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    return new_pos
+
+
+def plan(grid, grid_dimension:(int, int), current_pos):
+
+    layer = 0
+    pos = (0, 0)
+    start_pos = pos
+    plan =  []
+    orientation = FORWARD
```
Comment (elli-wolf) [inline hlc/plan/planner.py]: The current position, current layer, and orientation should be provided to the plan-function as parameters, so we call this function to re-plan too.

File: hlc/plan/planner.py [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+    return plan
+
+
+
+
+
+
+
+
```
Comment (elli-wolf) [inline hlc/plan/planner.py]: Why are here a bunch of empty lines :confused:

File: hlc/plan/planner.py [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+                else:
+                    pass
+
+
+    # The update of the plan array will either have to be added in the loop or in the move_one_unit_forward function.
+    plan = [(0, 1), (1, 1)]
+
+    return plan
+
+
```
Comment (elli-wolf) [inline hlc/plan/planner.py]: I think we can delete this part for now - @davidlkang will still be able to find it using Version Controll.

**Review by elli-wolf — CHANGES_REQUESTED** [review]

File: hlc/plan/planner.py (hunk 1 above)
Comment (selmaillig) [reply hlc/plan/planner.py:37]: that's the preparation for the obstacle avoidance function, but I'm gonna add returning the values 😬

**Review by elli-wolf — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**